src/HealthGPS.Input/model_parser.cpp

```diff
@@ -187,7 +187,9 @@ load_staticlinear_risk_model_definition(const nlohmann::json &opt, const Configu
     std::vector<core::DoubleInterval> policy_ranges;
     auto trend_models = std::make_unique<std::vector<LinearModelParams>>();
     auto trend_ranges = std::make_unique<std::vector<core::DoubleInterval>>();
+    auto trend_lambda = std::make_unique<std::vector<double>>();
     auto expected_trend = std::make_unique<std::unordered_map<core::Identifier, double>>();
+    auto expected_trend_boxcox = std::make_unique<std::unordered_map<core::Identifier, double>>();
```

Comment:
I know we talked about this on the last PR... but I'm still a bit confused as to why these need to be `std::unique_ptr`s. Even if they're being converted into `std::shared_ptr`s at some point can't you just `std::move` your `std::vector` (for example) into a `std::shared_ptr<std::vector<T>>`?

Reply:
Inside model parser, probably. I'll have a look.